providers/templates/providers/provider_detail.html

```diff
@@ -17,20 +17,23 @@
         <!-- Provider Details -->
         <div class="provider-info">
             <h2 class="provider-name">{{ provider.get_full_name }}</h2>
-            <p class="facility">Facility: {{ provider.facility|default:"Not Available" }}</p>
+            <p class="provider-role">Role: {{ provider.role }}</p>
```

Comment:
Why are we showing the role again in the provider detail view.
That is not needed.

Reply:
just deleted